### PR TITLE
[Confluent] Add examples for using --ids

### PR DESCRIPTION
**Pull request by fengzhou-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/confluent/azext_confluent/manual/_help.py [hunk 1]
```diff
@@ -15,3 +15,27 @@
...
 """
+
+helps['confluent organization show'] = """
+    type: command
+    short-summary: "Get the properties of a specific Organization resource."
+    examples:
+      - name: Show organization
+        text: |-
+               az confluent organization show --name "myOrganization" --resource-group "myResourceGroup"
+      - name: Show organization with ids
```
Comment (qwordy) [inline src/confluent/azext_confluent/manual/_help.py:26]:
```suggestion
      - name: Show organization using --ids
```

Comment (qwordy) [inline src/confluent/azext_confluent/manual/_help.py:26]:
```suggestion
      - name: Show organization using IDs
```

**Comment by qwordy** [comment]
Usage of --ids is global and trival. Do we need add examples for each command?

File: src/confluent/azext_confluent/manual/_help.py [hunk 2]
```diff
@@ -15,3 +15,27 @@
...
+"""
+
+helps['confluent organization delete'] = """
+    type: command
+    short-summary: "Delete Organization resource."
+    examples:
+      - name: Delete organization
+        text: |-
+               az confluent organization delete --name "myOrganization" --resource-group "myResourceGroup"
+      - name: Delete organization with ids
```
Comment (fengzhou-msft) [inline src/confluent/azext_confluent/manual/_help.py:38]:
```suggestion
      - name: Delete organization using IDs
```

**Review by houk-ms — APPROVED** [review]

**Comment by fengzhou-msft** [comment]
> Usage of --ids is global and trival. Do we need add examples for each command?

No requirements to do so for each command, but service team feels that this would help customers understand `--ids` better.

**Review by yungezz — APPROVED** [review]

**Pull request merged**